### PR TITLE
2344 validate encoding update

### PR DESCRIPTION
**Pull request by ghost**
Connects to bizneo/bizneo#2344

### CODE REVIEW

File: vendor/assets/javascripts/jquery.validationEngine-it.js [hunk 1]
```diff
@@ -1,118 +1,121 @@
...
+                    "regex": "none",
+                    "alertText": "* Fra ",
+                    "alertText2": " e ",
+                    "alertText3": " caratteri permessi"
+                },
+                "maxCheckbox": {
+                    "regex": "none",
+                    "alertText": "* Numero di caselle da selezionare in eccesso"
+                },
+				        "groupRequired": {
```
Comment (MamesPalmero) [inline vendor/assets/javascripts/jquery.validationEngine-it.js]: bad indentation

File: vendor/assets/javascripts/jquery.validationEngine-it.js [hunk 2]
```diff
@@ -1,118 +1,121 @@
...
+                    "alertText": "* URL non corretta"
+                },
+                "onlyNumber": {
+                    "regex": /^[0-9\ ]+$/,
+                    "alertText": "* Solo numeri"
+                },
+				        "onlyLetter": {
```
Comment (MamesPalmero) [inline vendor/assets/javascripts/jquery.validationEngine-it.js]: bad indentation

File: vendor/assets/javascripts/jquery.validationEngine-pt.js [hunk 3]
```diff
@@ -179,13 +179,17 @@
...
+		"dateTimeFormat": {
+		    "regex": /^\d{4}[\/\-](0?[1-9]|1[012])[\/\-](0?[1-9]|[12][0-9]|3[01])\s+(1[012]|0?[1-9]){1}:(0?[1-5]|[0-6][0-9]){1}:(0?[0-6]|[0-6][0-9]){1}\s+(am|pm|AM|PM){1}$|^(?:(?:(?:0?[13578]|1[02])(\/|-)31)|(?:(?:0?[1,3-9]|1[0-2])(\/|-)(?:29|30)))(\/|-)(?:[1-9]\d\d\d|\d[1-9]\d\d|\d\d[1-9]\d|\d\d\d[1-9])$|^((1[012]|0?[1-9]){1}\/(0?[1-9]|[12][0-9]|3[01]){1}\/\d{2,4}\s+(1[012]|0?[1-9]){1}:(0?[1-5]|[0-6][0-9]){1}:(0?[0-6]|[0-6][0-9]){1}\s+(am|pm|AM|PM){1})$/,
+		    "alertText": "* Data inválida ou mal formatada",
+		    "alertText2": "Formato esperado: ",
+		    "alertText3": "mm/dd/aaaa hh:mm:ss AM|PM ou ",
+		    "alertText4": "aaaa-mm-dd hh:mm:ss AM|PM"
+		},
+		"validate_encoding": {
+		    "regex": /^[\n -'*-@[-~£¥-¨ª¬®°²µ·¹-º½¿-ÂÄÆ-ÏÒ-ÔÖ-Üß-âäæ-ëí-ñó-ôö-ü˙˝Ω€]$/,
+		    "alertText": "* Caracteres inválidos presentes"
```
Comment (MamesPalmero) [inline vendor/assets/javascripts/jquery.validationEngine-pt.js:191]: isto não é o Português

File: vendor/assets/javascripts/jquery.validationEngine-zh_CN.js [hunk 4]
```diff
@@ -1,177 +1,177 @@
...
+                    "regex": "none",
+                    "alertText": "* 最少 ",
+                    "alertText2": " 个字符"
+                },
+                "maxSize": {
+                    "regex": "none",
+                    "alertText": "* 最多 ",
+                    "alertText2": " 个字符"
+                },
+				"groupRequired": {
```
Comment (MamesPalmero) [inline vendor/assets/javascripts/jquery.validationEngine-zh_CN.js]: bad indentation

File: vendor/assets/javascripts/jquery.validationEngine-zh_CN.js [hunk 5]
```diff
@@ -1,177 +1,177 @@
...
+                },
+                // --- CUSTOM RULES -- Those are specific to the demos, they can be removed or changed to your likings
+                "ajaxUserCall": {
+                    "url": "ajaxValidateFieldUser",
+                    // you may want to pass extra data on the ajax call
+                    "extraData": "name=eric",
+                    "alertText": "* 此名称已被其他人使用",
+                    "alertTextLoad": "* 正在确认名称是否有其他人使用，请稍等。"
+                },
+				"ajaxUserCallPhp": {
```
Comment (MamesPalmero) [inline vendor/assets/javascripts/jquery.validationEngine-zh_CN.js]: bad indentation

File: vendor/assets/javascripts/jquery.validationEngine-zh_CN.js [hunk 6]
```diff
@@ -1,177 +1,177 @@
...
+                    // remote json service location
+                    "url": "ajaxValidateFieldName",
+                    // error
+                    "alertText": "* 此名称可以使用",
+                    // if you provide an "alertTextOk", it will show as a green prompt when the field validates
+                    "alertTextOk": "* 此名称已被其他人使用",
+                    // speaks by itself
+                    "alertTextLoad": "* 正在确认名称是否有其他人使用，请稍等。"
+                },
+				 "ajaxNameCallPhp": {
```
Comment (MamesPalmero) [inline vendor/assets/javascripts/jquery.validationEngine-zh_CN.js]: bad indentation

File: vendor/assets/javascripts/jquery.validationEngine-zh_CN.js [hunk 7]
```diff
@@ -1,177 +1,177 @@
...
+                },
+	            //tls warning:homegrown not fielded
+                "dateFormat":{
+                    "regex": /^\d{4}[\/\-](0?[1-9]|1[012])[\/\-](0?[1-9]|[12][0-9]|3[01])$|^(?:(?:(?:0?[13578]|1[02])(\/|-)31)|(?:(?:0?[1,3-9]|1[0-2])(\/|-)(?:29|30)))(\/|-)(?:[1-9]\d\d\d|\d[1-9]\d\d|\d\d[1-9]\d|\d\d\d[1-9])$|^(?:(?:0?[1-9]|1[0-2])(\/|-)(?:0?[1-9]|1\d|2[0-8]))(\/|-)(?:[1-9]\d\d\d|\d[1-9]\d\d|\d\d[1-9]\d|\d\d\d[1-9])$|^(0?2(\/|-)29)(\/|-)(?:(?:0[48]00|[13579][26]00|[2468][048]00)|(?:\d\d)?(?:0[48]|[2468][048]|[13579][26]))$/,
+                    "alertText": "* 无效的日期格式"
+                },
+                //tls warning:homegrown not fielded
+				"dateTimeFormat": {
+	                "regex": /^\d{4}[\/\-](0?[1-9]|1[012])[\/\-](0?[1-9]|[12][0-9]|3[01])\s+(1[012]|0?[1-9]){1}:(0?[1-5]|[0-6][0-9]){1}:(0?[0-6]|[0-6][0-9]){1}\s+(am|pm|AM|PM){1}$|^(?:(?:(?:0?[13578]|1[02])(\/|-)31)|(?:(?:0?[1,3-9]|1[0-2])(\/|-)(?:29|30)))(\/|-)(?:[1-9]\d\d\d|\d[1-9]\d\d|\d\d[1-9]\d|\d\d\d[1-9])$|^((1[012]|0?[1-9]){1}\/(0?[1-9]|[12][0-9]|3[01]){1}\/\d{2,4}\s+(1[012]|0?[1-9]){1}:(0?[1-5]|[0-6][0-9]){1}:(0?[0-6]|[0-6][0-9]){1}\s+(am|pm|AM|PM){1})$/,
```
Comment (MamesPalmero) [inline vendor/assets/javascripts/jquery.validationEngine-zh_CN.js]: bad indentation

**Review by MamesPalmero — CHANGES_REQUESTED** [review]
@david-valero-bizneo Finished review

File: vendor/assets/javascripts/jquery.validationEngine-pt.js (hunk 3 above)
Comment (ghost) [reply vendor/assets/javascripts/jquery.validationEngine-pt.js:191]: Yes, it is :smile:

**Review by MamesPalmero — APPROVED** [review]
@david-valero-bizneo Acceptance passed

**Pull request merged**